cmd/contact-exporter/main.go

```diff
@@ -43,40 +46,20 @@ func (c contactExporter) findContacts() ([]string, error) {
 		return nil, err
 	}
 
-	contactMap := make(map[string]struct{}, len(contactsJSON))
-	for _, contactRaw := range contactsJSON {
-		var contactList []string
-		err := json.Unmarshal(contactRaw, &contactList)
-		if err != nil {
-			c.log.AuditErr(
-				fmt.Sprintf("couldn't unmarshal contact JSON %#v: %s\n",
-					contactRaw, err))
-			continue
-		}
-		for _, entry := range contactList {
-			// Only include email addresses
-			if strings.HasPrefix(entry, "mailto:") {
-				address := strings.TrimPrefix(entry, "mailto:")
-				// Using the contactMap to deduplicate addresses
-				contactMap[address] = struct{}{}
-			}
-		}
-	}
-
-	// Convert the de-dupe'd map back to a slice, sort it
-	var contacts []string
-	for contact := range contactMap {
-		contacts = append(contacts, contact)
-	}
-	sort.Strings(contacts)
-	return contacts, nil
+	return contactsList, nil
 }
 
-func writeContacts(contacts []string, outFile string) error {
-	data := []byte(strings.Join(contacts, "\n") + "\n")
+// The `writeContacts` function produces a file containing JSON serialized
+// contact objects
+func writeContacts(contactsList []contact, outfile string) error {
+	data, err := json.Marshal(contactsList)
+	if err != nil {
+		return err
+	}
+	data = append(data, '\n')
```

Comment:
Since `vim` and other editors are likely to introduce this on their own I lean on the side of putting it there to be a good plaintext citizen.

@jsha Do you want to be the tie breaker? :-)